### PR TITLE
feat(client): Add mysql fulltext search tests

### PR DESCRIPTION
**Pull request by garrensmith**
Part of https://github.com/prisma/prisma/issues/10415

### CODE REVIEW

File: packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  })
+
+  /**
+   * Test search with OR
+   */
+  test('findMany with OR', async () => {
+    const result = await prisma.user.findMany({
+      where: {
+        name: {
+          search: 'John April',
```
Comment (Jolg42) [inline packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts:84]: Oh so `OR` is a blank space?

File: packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+          name: John Pearl,
+        },
+      ]
+    `)
+  })
+
+  /**
+   * Test search with the `-` operators
+   */
+  test('findMany with & and |', async () => {
```
Comment (Jolg42) [inline packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts:113]:
```suggestion
  test('findMany with -', async () => {
```

**Review by Jolg42 — APPROVED** [review]
Looks good (see comments)

Would be good to rename https://github.com/prisma/prisma/blob/mysql-fts/packages/client/src/__tests__/integration/happy/full-text-search directory to `full-text-search-postgresql` I think

File: packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts (hunk 1 above)
Comment (Jolg42) [reply packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts:84]: So https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html

Comment (garrensmith) [reply packages/client/src/__tests__/integration/happy/full-text-search-mysql/test.ts:84]: Yes. Its a lot more basic thank what Postgres offers

**Review by yazdanidana — COMMENTED** [review]
danayzdanyyzdanydana@gmail.com

**Comment by Jolg42** [comment]
I see the rename makes the snapshot test fail, not the best DX on this one 😓

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**